build/Tests/UnitTests/CoreGraphics.Drawing/CoreGraphics.Drawing.UnitTests.vcxproj

```diff
@@ -237,6 +237,7 @@
     </ClangCompile>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClangCompile Include="$(StarboardBasePath)\tests\unittests\CoreGraphics.drawing\CGContextDrawing_AntiAliasTests.cpp" />
```

Comment:
nit: move this down with its friends. The entry point and framework are here, the tests are below.